src/api/bitswap.js

```diff
@@ -1,29 +1,9 @@
 'use strict'
 
-const promisify = require('promisify-es6')
-
-module.exports = (send) => {
+module.exports = (arg) => {
```

Comment:
Initially I had some problems when I structured it like you are saying. But I don´t remember what was the problem. I will try again 😃